### PR TITLE
Add alpine php7-min image

### PR DESCRIPTION
**Pull request by cyberhck**
- add 61.4MB alpine php7 image

### CODE REVIEW

**Comment by sergeyklay** [comment]
Could you please separate this PR?

**Comment by cyberhck** [comment]
done (with a lot of issues, accidently branched off from one branch instead of master)

**Comment by sergeyklay** [comment]
Thank you

File: php/alpine-php7-min/Makefile [hunk 1]
```diff
@@ -0,0 +1,27 @@
+NAME = phalconphp/php
+TAG = alpine-php7-min
```
Comment (sergeyklay) [inline php/alpine-php7-min/Makefile]: TAG `alpine-php7-min`
Folder `alpine-php7-min`
Dockerfile for `phalconphp/php:alpine-3-php7`

**Comment by sergeyklay** [comment]
https://github.com/phalcon/dockerfiles/commit/4b507eaedc1711282aea679bc3d1c78868109716

**Pull request merged**